signals/commands/policies/src/main/java/org/eclipse/ditto/signals/commands/policies/modify/ModifyPolicy.java

```diff
@@ -105,10 +106,16 @@ public static ModifyPolicy fromJson(final String jsonString, final DittoHeaders
      */
     public static ModifyPolicy fromJson(final JsonObject jsonObject, final DittoHeaders dittoHeaders) {
         return new CommandJsonDeserializer<ModifyPolicy>(TYPE, jsonObject).deserialize(() -> {
-            final String policyId = jsonObject.getValueOrThrow(PolicyModifyCommand.JsonFields.JSON_POLICY_ID);
+            final String policyId;
             final JsonObject policyJsonObject = jsonObject.getValueOrThrow(JSON_POLICY);
             final Policy policy = PoliciesModelFactory.newPolicy(policyJsonObject);
 
+            final Optional<String> optionalPolicyId = jsonObject.getValue(PolicyModifyCommand.JsonFields
+                    .JSON_POLICY_ID);
+            policyId = optionalPolicyId.orElseGet(() -> policy.getId().orElseThrow(() ->
```

Comment:
why not declare the variable in this line like `final String policyId= ...`?
Please change